pkg/gen/nodejs-templates/kind.ts.mustache

```diff
@@ -85,7 +85,6 @@ import { getVersion } from "../../version";
           {{#Aliases}}
               { parent: opts.parent, type: "{{.}}", name: name },
           {{/Aliases}}
-              ...((opts && opts.aliases) || []),
```

Comment:
It looks like this will cause us to drop any user-specified aliases. I don't think we want that. Did you try changing `((opts && opts.alaises) || [])` to `...(opts.aliases || [])`?

Reply:
```        opts.aliases = [
            { parent: opts.parent, type: "kubernetes:apps/v1:Deployment", name: name },
            { parent: opts.parent, type: "kubernetes:apps/v1beta1:Deployment", name: name },
            { parent: opts.parent, type: "kubernetes:apps/v1beta2:Deployment", name: name },
            { parent: opts.parent, type: "kubernetes:extensions/v1beta1:Deployment", name: name },
            ...(opts.aliases || []),
        ];
```
still causes a panic.

Reply:
Ah, I'll bet know what the issue is. We're mutating an object that is passed in by the caller. We shouldn't do that. In the case of the original bug, I think we're adding aliases for multiple resources to the same `opts` dictionary.

Reply:
Yep, you're right. I've updated the PR to fix. Looks like we were already handling this properly in the Python SDK.